src/cljs/rems/table.cljs

```diff
@@ -18,16 +18,21 @@
 (defn column-class [column-definitions col]
   (get-in column-definitions [col :class] (name col)))
 
+(defn column-class-customized [item column-definitions col]
+  (if (= "draft" (:state item))
```

Comment:
This sort of logic shouldn't be part of a "generic table widget" (see ns docstring)

I think this should be done either by
a) using a `[:span {:class "highlight-draft"} ...]` as the value of a column
b) adding a :class fn to the column definition (just like we have the :value fn)